content/release-10-15-0/devices-10-15-0-bundle/10-15-0-262.md

```diff
@@ -0,0 +1,54 @@
+---
+weight: 28
+title: Release 10.15.0.262
+layout: bundle
+---
+
+<!--10.15.0.250 - 10.15.0.262-->
+
+
+### Fixes
+
+<div><table ><colgroup>
+<col style="width: 15%;"><col style="width: 70%;"><col style="width: 15%;"></colgroup>
+<thead><tr>
+<th>
+Component</th>
+<th>
+Description</th>
+<th>
+Issue</th>
+</tr>
+</thead><tbody>
+
+<tr>
+<td>Device Management</td>
+<td>The "Groups" column in the device list used to display all groups higher in the hierarchy to which the device belonged directly or transitively. This has been fixed and now only the direct parent groups are displayed.</td>
+<td>DM-1870</td>
+</tr>
+
+<tr>
+<td>Device Management</td>
+<td>The groups list on the <b>Groups</b> page now allows to filter by the grid's columns.</td>
+<td>DM-1836</td>
+</tr>
+
+<tr>
+<td>Device Management</td>
+<td>By adding or removing an external identifier of type <code>c8y_Serial</code> to/of the monitored device, the device connection monitoring is notified accordingly to start device connection monitoring or to stop device connection monitoring with the specified external identifier.</td>
+<td>MTM-50025</td>
+</tr>
+
+<tr>
+<td>Device Management</td>
+<td>An issue with the filter functionality on the <b>Groups</b> page for smart groups has been fixed. Results now include devices belonging to the selected smart groups.</td>
+<td>DM-1872</td>
+</tr>
+
+<tr>
+<td>MQTT</td>
+<td>The MQTT storage is now properly cleared by removing not needed data after device disconnection. This may result in performance improvements, especially in terms of MQTT connect and subscribe operations.</td>
```

Comment:
```suggestion
<td>The MQTT storage is now properly cleared by removing not needed data after a device has disconnected. This may result in performance improvements, especially in terms of MQTT connect and subscribe operations.</td>
```